### PR TITLE
Use local SSL CA bundle

### PR DESCRIPTION
**Pull request by tsujigiri**
We found Adk2's CA certs to be missing on several machines. This adds them to
the repository and makes the client use these instead of the ones installed on
the respective machine.

### CODE REVIEW

File: lib/adk2/client.rb [hunk 1]
```diff
@@ -29,17 +29,15 @@ def report(level, name, start_date, end_date)
 
     def get_api_token
       return @api_token = nil if username.nil? || password.nil?
-      response = RestClient.post(
-        api_base_url + '/auth',
+      response = resource('auth').post(
         {
           'email' => username,
```
Comment (dziemba) [inline lib/adk2/client.rb]: You can use symbols as keys here

**Comment by dziemba** [comment]
:+1:

File: lib/adk2/client.rb (hunk 1 above)
Comment (tsujigiri) [inline lib/adk2/client.rb]: So? :sunglasses:

Comment (dziemba) [inline lib/adk2/client.rb]: Looks nicer and is faster :ghost:

Comment (tsujigiri) [inline lib/adk2/client.rb]: Benchmark says no. :P

Comment (dziemba) [inline lib/adk2/client.rb]: You knew I was kidding about the faster thing ;)

**Pull request merged**